### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by tiepinuanhu** [massCodeIO/massCode#284] [Q&A]
Why is the top right corner of my MassCode interface displayed unsponsored

**Comment by andercard0** [comment]
> Why is the top right corner of my MassCode interface displayed unsponsored

Because you not **sponsored it**!

Although to be a free application, and a one man project, sponsors can show their gratitude to the developer efforts paying him any ammount of money. Doing so, you will have not see that tag anymore! Also, this is not a problem as you label it in your title!
